code/src/containers/forms/product-form/components/additional-info/AdditionalInfo.tsx

```diff
@@ -0,0 +1,110 @@
+
+import AppBox from "@/components/app-box/AppBox";
+import AppCheckbox from "@/components/app-checkbox/AppCheckbox";
+import AppFormHelperText from "@/components/app-form-helper-text/AppFormHelperText";
+import AppInput from "@/components/app-input/AppInput";
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppSelect from "@/components/app-select/AppSelect";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+const AdditionalInfo = ({
+  register,
+  errors,
+  control
+}: ProductFormAdditionalInfoSectionProps) => {
+  const categoryErrorElement = errors.category && (
+    <AppFormHelperText className="product-form__category-select-helper" error>
+      {errors.category.message}
+    </AppFormHelperText>
+  );
+
+  const selectControllerRenderFunction = ({
+    field: handlers
+  }: {
+    field: ControllerRenderProps<ProductFormValues>;
+  }) => (
+    <>
+      <AppSelect
+        fullWidth
+        label="productForm.inputLabel.category"
+        inputProps={{
+          className: "product-form__category-select"
+        }}
+        error={Boolean(errors.category)}
+        data-testid="product-form-category-select"
+        {...handlers}
+      >
+        {productCategories.map((item) => (
+          <AppMenuItem value={item.id} key={item.id}>
+            <AppTypography
+              className="product-form__category-select-label"
+              translationKey={item.label}
+            />
+          </AppMenuItem>
+        ))}
+      </AppSelect>
+      {categoryErrorElement}
+    </>
+  );
+
+  return (
+    <AppBox className="product-form__container product-form__additional-info-section">
+      <AppBox className="product-form__header">
+        <AppTypography
+          component="h1"
+          variant="h3"
+          translationKey="productForm.section.additionalInformation.title"
+          className="product-form__header-title"
+        />
+      </AppBox>
+      <AppBox className="product-form__body">
+        <AppBox className="product-form__quantity-price-container">
+          <AppInput
+            fullWidth
+            labelTranslationKey="productForm.inputLabel.price"
+            inputProps={{ min: 0, step: "any" }}
+            type="number"
+            error={Boolean(errors.price)}
+            helperText={errors.price ? errors.price.message : undefined}
+            data-testid="product-form-price-input"
+            {...register("price", { valueAsNumber: true })}
+          />
+          <AppInput
+            fullWidth
+            labelTranslationKey="productForm.inputLabel.quantity"
+            inputProps={{ min: 0 }}
+            type="number"
+            error={Boolean(errors.quantity)}
+            helperText={errors.quantity ? errors.quantity.message : undefined}
+            data-testid="product-form-quantity-input"
+            {...register("quantity", { valueAsNumber: true })}
+          />
+        </AppBox>
```

Comment:
We can also try to extract common props for inputs, move them to object and spread. What do you think?

Reply:
If you mean creating of one object with shared properties, I think there is no much sense because those components have only two properties with the same value